backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po

```diff
@@ -59,28 +59,24 @@ msgstr ""
 #: tracim_backend/applications/share/config.py:22
 msgid ""
 "[{website_title}] You shared \"{content_filename}\" with {nb_receivers} "
-"users"
+"people"
 msgstr ""
 "[{website_title}] Vous avez partagé  \"{content_filename}\" avec "
-"{nb_receivers} utilisateurs"
+"{nb_receivers} personnes"
 
 #: tracim_backend/applications/upload_permissions/config.py:14
-msgid ""
-"[{website_title}] {emitter_name} allowed you to upload files in this "
-"shared space \"{workspace_number}\""
-msgstr ""
-"[{website_title}] {emitter_name} vous a autorisé à téléverser des "
-"fichiers dans l'espace partagé \"{workspace_number}\""
+msgid "{emitter_name} invited you to upload files on \"{website_title}\""
+msgstr "{emitter_name} vous a invité à téléverser fichiers sur \"{website_title}\""
 
-#: tracim_backend/applications/upload_permissions/config.py:23
+#: tracim_backend/applications/upload_permissions/config.py:21
 msgid ""
-"[{website_title}] You allowed {nb_receivers} users to upload files on "
+"[{website_title}] You invited {nb_receivers} people to upload files on "
 "\"{workspace_name}\""
 msgstr ""
-"[{website_title}] Vous avez autorisé {nb_receivers} utilisateurs à "
-"téléverser des fichiers dans \"{workspace_name}\""
+"[{website_title}] Vous avez invité {nb_receivers} personnes à téléverser "
```

Comment:
```suggestion
"[{website_title}] Vous avez invité {nb_receivers} personne(s) à téléverser "
```

Reply:
I suggest this to handle the share to one people case.

Reply:
In this case, check everywhare to add `(s)` when it's necessary.

Reply:
agreed

Reply:
done.

Reply:
done.